cri/config/config.go

```diff
@@ -1,5 +1,12 @@
 package config
 
+const (
+	// K8sNamespace is the namespace we use to connect containerd when CRI is enabled.
+	K8sNamespace = "k8s.io"
+	// ContainerdAddr is the default unix socket address of containerd when CRI is enabled.
+	ContainerdAddr = "/var/run/containerd/containerd.sock"
```

Comment:
Will this part in CRI directly communicate with containerd? 
If that we need to do a detailed plan to set up a long roadmap of this.
If this is a temporary work?
If we need to implement this is container manager first?
And something others.

Reply:
@allencloud Actually, we need to do a detailed plan. : )

Reply:
Talked with you offline, and I think we need update of this Part.